book/src/exec.md

```diff
@@ -0,0 +1,168 @@
+# Taiga execution model
+
+## On a high level
+**Intents** are a mechanism that allows users to enact a multiparty state transition. Users express their interests with intents, and Taiga makes sure that the intents are satisfied atomically.
+
+Here is the high level description of the process, starting from creating intents and ending up publishing a blockchain transaction representing the desired state transition:
+1. **Create**: A user creates an intent, publishing the information of what they have and what they want
+2. **Gossip**: The intent goes to the intent gossip network and is gossiped around the solver nodes
+3. **Solve**: Solvers find matching intents and update the intent with partial transactions until the intent is fully satisfied
+4. **Finalize**: When the intent is satisfied, a transaction is created and published on the blockchain
+
+![exec_high.png](img/exec_high.png)
+
+## Step 1: Create an intent
+
+Intents specify ephemeral interests of users. The step of intent creation consists of two parts:
+1. **Specify an intent**. `IntentVP` encodes the user interests in the [VP](./validity-predicates.md) form. As in a valid transaction all VPs must evaluate to `True`, the intent is guaranteed to be satisfied.
```

Comment:
We should clarify what "intent VP" is - in my understanding intent VP is not a distinction in the code, only a distinction in usage, right?

Reply:
Not a distinction from other VPs or from intents?

Reply:
We have send/receive VPs on notes, right? And an intent VP is a usage of the send VP to encode the desires of a user (also receives X...)? (perhaps my understanding is out of date)